docs/source/tutorials/QUBO_problem.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>the running time must be not good due to the nonjit implementation for the cvar. in the long term, better refactor the code in applications/optimization to support jit for cvar</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>